templates/org/aac/aac-browser-header.yml

```diff
@@ -2,10 +2,10 @@
 metadata:
   title: <Article title, which becomes the title metadata>
   titleSuffix: Azure Architecture Center
-  description: <Write a 100-160 character description that ends with a period and starts with a call to action. This becomes the browse card description.>
-  author: <Contributor's GitHub username. If no GitHub account, use martinekuan>
-  ms.author: <Contributor's Microsoft alias. Can include multiple contributors, separated by commas. If there's no alias, use the Microsoft email alias "architectures".>
-  ms.date: <Publish or major update date - mm/dd/yyyy>
+  description: <Write a 100-160 character description that starts with a call to action and ends with a period. This becomes the browse card description.>
+  author: <Contributor's GitHub username>
+  ms.author: <Contributor's Microsoft alias. This can include multiple contributors, separated by commas.>
+  ms.date: <Date of publish or major update, in mm/dd/yyyy format>
   ms.topic: conceptual
   ms.service: architecture-center
   ms.subservice: example-scenario
```

Comment:
We can't have this hard coded -- this is what this work item was about.  We need to take this header stuff and either put it into each article OR we need to have this be a "choose one" option here.  This is used for solution idea, example workload, and reference architecture -- they all can't be `example-scenario`.